packages/stream/index.d.ts

```diff
@@ -68,15 +71,15 @@ declare module '@nivo/stream' {
         fillOpacity: number
 
         borderWidth: number
-        borderColor: InheritedColorConfig
+        borderColor: InheritedColorConfig<T>
 
         enableDots: boolean
         renderDot: StreamDotsItem
         dotPosition: 'start' | 'center' | 'end'
         dotSize: DatumToNumber | number
-        dotColor: InheritedColorConfig
+        dotColor: InheritedColorConfig<T>
```

Comment:
I think custom accessors for the dots will receive the following structure:

```ts
interface DotDatum {
    key: string
    id: number
    color: string
    index: number
    x: number
    value: number
    y1: number
    y2: number
}
```

Apparently `Datum`, in the definitions file.

It's a combination of [what's computed for the layer points](https://github.com/plouc/nivo/blob/master/packages/stream/src/hooks.js#L91), and some extra properties passed in https://github.com/plouc/nivo/blob/master/packages/stream/src/StreamDots.js#L35.